{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -20,21 +25,33 @@ classifiers = [
 [project.optional-dependencies]
 # here you can add dependency groups like this: 
 # group = [dep1, dep2,...]
+doc = []
+dev = [
+    "black",            # code formatting
+    "ruff",             # code linting
+    "mypy",             # static type checking
```

Comment:
including mypy here is fine, but the template isn't curently steup to use it in anyway so the people making the plugin will have to use it manually. I also don't know if it will throw warnings or errors out of the box. Given that Hydromt's support for mypy is spotty at best the chance is fairly high, so not 100% yet on whether we want to include this.

Reply:
I do not it (was a Copilot suggestion) so I leave it to you to decide if we include it or not :)